### ISSUE TITLE
Proposals registration fails

### ISSUE THREAD

**Issue by Peter-SovietSquirrel** [mysteriumnetwork/node#2180]
Node is not visible in app but is online and connectable 

DAEMON_OPTS="--experiment-natpunching=false --tequilapi.address=127.0.0.1 --p2p.listen.ports=51820:52075"
SERVICE_OPTS="--openvpn.port=10112 --wireguard.listen.ports=52820:53075 openvpn,wireguard"

 - Node version: 0.31.2
 - Rasp stretch
 - App ver. 0.31.2

This hasn't got anything to do with price setting for consumer. Tested with new identity, same result.
Emailed log.

**Comment by etherunit** [comment]
Thanks Peter! I am adding your node logs here so our engineering team can look through.

My node is also not visible in app. Doing proposals 0x409b5c902d26d4c5250dce8cbd574bd0cbe2b8cb in CLI shows 0 proposals available.
In both versions: 0.31.1 & 0.31.2

**Comment by Mariskas80** [comment]
Good Morning
yesterday I updated the node to the new version and it stopped appearing in the android application listing.
on the dashboard everything is fine.

**Comment by Waldz** [comment]
@Peter-SovietSquirrel we see that your service was started with very high prices 0.5 myst/GB and 0.001 myst/min

Please check your config.yml file if you override price settings, because our Android app currently is excluding very high prices.

```
2020-05-07T00:51:23.434 DBG source/config/config.go:177              > Returning user config value wireguard.price-gb:0.5
2020-05-07T00:51:23.436 DBG source/config/config.go:177              > Returning user config value wireguard.price-minute:0.001
```

**Comment by Mariskas80** [comment]
it's true ... I've been exploring the news :)
meanwhile I put it back to zero, but the problem continues :(
what should I do?

**Comment by Waldz** [comment]
Either:
- (recommended) deleting `price-minute` and `price-gb` lines in config.yml will leed to correct default pricing
- set those lines to default pricing manually

```
myst help service
...
   --openvpn.price-minute value      Sets the price of the OpenVPN service per minute. (default: 0.0005)
   --openvpn.price-gb value          Sets the price of the OpenVPN service per minute. (default: 0.07)
  ... 
  --wireguard.price-minute value    Sets the price of the wireguard service per minute. (default: 0.0005)
  --wireguard.price-gb value        Sets the price of the wireguard service per minute. (default: 0.07)
```

**Comment by Mariskas80** [comment]
problem solved... i put default values 0.0005 and 0.07... save settings... restart server... and voila!!!
thank you very much

**Comment by Waldz** [comment]
> My node is also not visible in app. 0x409b5c902d26d4c5250dce8cbd574bd0cbe2b8cb
In both app versions: 0.31.1 & 0.31.2

Similar issue here, just service was started with 0 price (Android skips this pricing)
Service restarts in UI leads to 0 price, fixed in #2116 (will be released in 0.32.0)
